cli/crates/common/src/analytics.rs

```diff
@@ -64,8 +57,8 @@ impl Analytics {
 
         let analytics_data_and_variables = write_key
```

Comment:
IMHO it would be more readable to just have:
```
write_key.zip(dataplan_url)
  .filter(|_| do_not_track.is_none() && data.map(|data| data.enable_analytics).unwrap_or_default())
```
Rather than having `invert()` and `zip()`. Especially that `zip(data.filter(|data| !data.enable_analytics).invert());` is really hard to read.